success-page.php

```diff
@@ -0,0 +1,7 @@
+<?php include 'includes/header.php'; ?>
```

Comment:
Style this page further please, text looks weird left aligned whilst everything else is centred.

Reply:
done